.taplo.toml

```diff
@@ -0,0 +1,11 @@
+# See https://taplo.tamasfe.dev/configuration/file.html
+# and https://taplo.tamasfe.dev/configuration/formatter-options.html
+
+[formatting]
+# This is a matter of taste, but expanded inline arrays make tables harder to read.
+array_auto_collapse = true
```

Comment:
This seems to already be the default

Reply:
Yeah not much point in defining `array_auto_collapse` here as it matches the default.